### PR TITLE
G602 now uses the G604 svg

### PR DESCRIPTION
**Pull request by dhollinger**
After the amazing work that @markusressel did with the G604, I took that file and made some slight modifications to create an svg for the G602.

I would venture to guess that this PR is dependent on https://github.com/libratbag/libratbag/pull/838

### CODE REVIEW

**Comment by markusressel** [comment]
Is the SVG file any different than the one from the G604? Because if it is not I think they use symlinks instead of copying the file.

**Comment by dhollinger** [comment]
@markusressel It is. I was going to symlink it, but the G602 swaps G7 with G9, G4 with G6, and the `+` and `-` buttons are G10 and G11 respectively

File: data/meson.build [hunk 1]
```diff
@@ -51,6 +51,7 @@ svgs = [
 	'svgs/logitech-g500s.svg',
 	'svgs/logitech-g502.svg',
 	'svgs/logitech-g600.svg',
+    'svgs/logitech-g602.svg',
```
Comment (whot) [inline data/meson.build]: check the indentation on this one please

File: data/svgs/README.md [hunk 2]
```diff
@@ -82,6 +82,7 @@ as of the time of import. See the libratbag COPYING file for details.
  logitech-g500s.svg
  logitech-g502.svg
  logitech-g600.svg
+ logitech-g602.svg
```
Comment (whot) [inline data/svgs/README.md]: this list doesn't need to be updated, it's just the list of devices imported from libratbag which doesn't apply here

**Comment by whot** [comment]
> I would venture to guess that this PR is dependent on libratbag/libratbag#838

fwiw that is true for device support but they're independent otherwise, i.e. we can merge this into piper without having to wait for libratbag.

**Comment by dhollinger** [comment]
@whot It should be good to go now. After that PR was closed with the explanation, I updated the device id in this PR to match what's in libratbagd.

**Comment by whot** [comment]
merged, thanks!

**Pull request merged**